pylint/checkers/classes/class_checker.py

```diff
@@ -127,6 +127,62 @@ def _definition_equivalent_to_call(definition, call):
     return all(kw in call.args or kw in definition.kwonlyargs for kw in call.kws)
 
 
+def _is_trivial_super_delegation(function: nodes.FunctionDef) -> bool:
+    """Check whether a function definition is a method consisting only of a
+    call to the same function on the superclass.
+    """
+    if (
+        not function.is_method()
+        # With decorators is a change of use
```

Comment:
I'm not sure I understand what you mean with this comment. Could you be a little bit more specific?